2-mandatory/5-reading-list.js

```diff
@@ -15,11 +15,13 @@ You should modify the tests so that they contain the values that correspond to y
 In this style of testing it is typical to write out as strings exactly what you expect your output to be, 
 without using any variables or any logic like loops, template strings or if statements.
 */
-
+// I DO NOT GET THIS
 const books = [];
   
 // exercise 1
 function logBooks() {
+  books.forEach((book) => console.log(`${book.title} by ${book.author}`));
```

Comment:
This function looks good, just needs the creation of the objects in the books array on line 19 so it has books to iterate through when called.